subprojects/wrapper/src/crossVersionTest/groovy/org/gradle/integtests/WrapperCrossVersionIntegrationTest.groovy

```diff
@@ -104,6 +104,7 @@ task hello {
     }
 }
 """
+        settingsFile << ''
```

Comment:
⭕️ Is this still necessary after you introduced the property? (same for the other occurrences above)

Reply:
I am going to go through them one by one again and will leave the reasoning here:

- `ToolingApiIntegrationTest."tooling API does not hold JVM open"`: We run a build with the `GradleExecutor` which in turn runs a Java main class. That main class uses the Tooling API to execute a task of the same build script (which doesn't have a settings file) and will search upward. The alternative is to use `((DefaultGradleConnector) connector).searchUpwards(false);` here which might make the intention more clear. Changed to that.